django_cte/query.py

```diff
@@ -10,9 +10,7 @@
 class CTEQuery(Query):
     """A Query which processes SQL compilation through the CTE compiler"""
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self._with_ctes = []
+    _with_ctes = ()
```

Comment:
Can you elaborate on the motivation for switching from a list to a tuple?

Reply:
Yes, it follows the same pattern as `combined_queries`, which is also a tuple.